http-verbs-common/src/main/scala/uk/gov/hmrc/play/http/logging/MdcLoggingExecutionContext.scala

```diff
@@ -17,19 +17,11 @@
 package uk.gov.hmrc.play.http.logging
 
 import org.slf4j.MDC
-import play.api.libs.concurrent.Execution.defaultContext
-import uk.gov.hmrc.http.logging.LoggingDetails
 
-import scala.collection.JavaConverters._
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
-@deprecated("MdcLoggingExecutionContext no longer required, please inject Play's default EC instead", "8.9.0")
```

Comment:
We should add this removal, and the alternative to the README/CHANGELOG if not already there.